base/base/wide_integer_impl.h

```diff
@@ -32,6 +34,59 @@ namespace CityHash_v1_0_2 { struct uint128; }
 namespace wide
 {
 
+constexpr bool supportsBitInt256()
+{
+#if defined(__x86_64__)
+    return true;
+#else
+    return false;
```

Comment:
In theory, Clang supports `_BitInt` across all platforms. Maybe you could do one CI run where `supportsBitInt256` returns `true` and see what platforms dont' work yet.

Reply:
I tried before, it is not supported in arm.